src/fileformats/ocd_file_import.cpp

```diff
@@ -1131,6 +1140,39 @@ void OcdFileImport::importLayoutObjects(const QString& param_string)
 			if (ok)
 				object_number = i_value;
 			break;
+		case 'x':
```

Comment:
I know that these parameters are shared with template image import. That said, is there a chance of introducing a shared function for handling the x, y, a, b, u and v parameters?